internal/exec/stages/files/files_test.go

```diff
@@ -29,10 +28,6 @@ func (pw pathWrapper) getPath() string {
 	return string(pw)
 }
 
-func (pw pathWrapper) create(l *log.Logger, u util.Util) error {
```

Comment:
This is an odd function to accidentally become unused.  It turns out it's trying to fulfill the `filesystemEntry` interface, which changed in fd6103a5d7aef884377748b611596be4b5893d44.  We should figure out what this test is trying to accomplish and fix it.

Reply:
```
internal/exec/stages/files/files_test.go:32:23: func `pathWrapper.create` is unused (unused)
```
Yeah, I agree with you. I was under the impression that this function is just returning a `nil`, and the linter caught it as an unused entry so thought of dropping it. I'm still confused with the `unused` vs `deadcode` warnings.

`unused: Checks Go code for unused constants, variables, functions and types`
`deadcode: Finds unused code`